proto/decentraland/sdk/components/light_source.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+package decentraland.sdk.components;
+import "decentraland/sdk/components/common/id.proto";
+import "decentraland/common/colors.proto";
+import "decentraland/common/texture.proto";
+option (common.ecs_component_id) = 1079;
+
+message PBLightSource {
+  optional bool active = 1;                                           // default = true, whether the lightSource is active or not.
+  optional decentraland.common.Color3 color = 2;                      // default = Color.white, the tint of the light, in RGB format where each component is a floating point value with a range from 0 to 1.
+  optional float intensity = 3;                                       // default = 250, ranges from 1 (dim) to 100,000 (very bright), expressed in Lumens for Point and Spot.
```

Comment:
not sure if 100k max is enough for lumens; in our version the sun at midday is around 125,000 lumens equivalent (10k l/m^2). you would need a point/spot light to be brighter to have the same effect at a range greater than 1m away.

creators can always cheat if they want something brighter by putting multiple lights at the same location, so it is not effective at preventing brighter lights anyway.

Reply:
In real life, if you turn on a lightbulb outside at mid-day you won't see any effects as well, IMO that's reasonable
That's why we made sure we added the control over the skybox time before we did dynamic lights, so that creators can make sure they have a night-time sky for their lights
I think our reference measurements should be with the midnight ambient light, not the midday light

Reply:
sure, but do we need a maximum?

dani mentioned it is not actually capped anyway.